### PR TITLE
Added ROADMAP file

### PR DESCRIPTION
**Pull request by scottinet**

### CODE REVIEW

**Comment by ballinette** [comment]
+1

**Comment by j33f** [comment]
+1

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+## Won't do
+
+*This section is currently empty*
+
+## Done
+
+* Plugins configuration through the command-line interface (see [PR #253](https://github.com/kuzzleio/kuzzle/pull/253))
+* Added administrative functions to the command-line interface (see [PR #208](https://github.com/kuzzleio/kuzzle/pull/208))
```
Comment (EmilieEsposito) [inline ROADMAP.md]: Inconsistency in tense.
Should be something like "more administrative functions..."

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Allow users to store data in memory storage using the API (see [PR #224](https://github.com/kuzzleio/kuzzle/pull/224))
+* Protect Kuzzle servers against flooding (see [PR #217](https://github.com/kuzzleio/kuzzle/pull/217))
+* Allow listener plugins to be multithreaded (see [PR #237](https://github.com/kuzzleio/kuzzle/pull/237))
+* Updated ElasticSearch to v2 (see [PR #201](https://github.com/kuzzleio/kuzzle/pull/201))
```
Comment (EmilieEsposito) [inline ROADMAP.md]: Inconsistency in tense.

**Comment by EmilieEsposito** [comment]
+1

**Comment by EmilieEsposito** [comment]
Do not merge until @melmag has given her approval

**Comment by j33f** [comment]
first merged PR for @melmag \o/

**Pull request merged**